controller/src/main/java/io/pravega/controller/store/stream/Stream.java

```diff
@@ -236,22 +236,22 @@
     /**
      * Commits a transaction
      * If already committed, return TxnStatus.Committed
-     * If aborted, throw OperationOnTxNotAllowedException
+     * If aborted, throw IllegalStateException
```

Comment:
This is not a checked exception so we don't this in the declaration. Also there are other exceptions this method can throw so this declaration doesn't seem right. Have retained the javadoc though.